### DISCUSSION TITLE
Failed Login Attempt Limiting

### DISCUSSION THREAD

**Discussion by cmhardage** [datalust/seq-tickets#1804] [Help] [answered]
My organization is evaluating Seq. We are running an Individual instance of Seq 2023.1.9101 in a Docker container. I enabled authentication and created a single Administration role user. During a penetration test our security team ran against the server they observed that Login page allows multiple failed attempts. They've flagged this as a potential avenue for brute force attacks. 

I reviewed the documentation and have searched the Discussions but am unable to find a setting that sets a limit on failed login attempts. Does this setting exist and I've missed it? Is it a feature in Team, Enterprise, or Datacenter?

Thanks in advance for any information you can provide!

**Comment by liammclennan** [comment, marked as answer]
Hi @cmhardage 

Seq supports password complexity requirements and uses a computationally expensive password verification algorithm that makes a brute force attack impractical, even without limiting login attempts. If Seq did implement limiting of login attempts then it would create a denial-of-service risk. 

To have full control over authentication, as well as features like multi-factor authentication, you can configure Seq to use Azure Active Directory or an OpenID Connect provider.
